package.json

```diff
@@ -6,7 +6,28 @@
   "scripts": {
     "start": "node app",
     "build": "webpack && node app @apostrophecms/asset:build",
-    "dev": "node app @apostrophecms/asset:build && webpack serve & ENV=dev node app"
+    "dev": "node app @apostrophecms/asset:build && webpack serve & ENV=dev nodemon app"
+  },
+  "nodemonConfig": {
+    "verbose": true,
+    "watch": [
+      "./app.js",
+      "./modules/**/*",
+      "./lib/**/*.js",
+      "./views/**/*.html",
+      "./src/**/*"
```

Comment:
Isn't `src` the province of webpack hot reload rather than nodemon? This would cause an unnecessary server restart along with the frontend hot reload, no?

Reply:
that's probably right. We should remove it from the demo as well.